bundle/bundle_test.go

```diff
@@ -214,17 +214,18 @@ func TestValuesOrDefaults_Required(t *testing.T) {
 }
 
 func TestValuesOrDefaults_NotApplicableToAction(t *testing.T) {
-	is := assert.New(t)
+	// vals represent user-supplied parameter values
 	vals := map[string]interface{}{
 		"param-with-default-and-override": true,
 	}
+
 	b := &Bundle{
 		Definitions: map[string]*definition.Schema{
 			"param-with-default-not-applicable": {
 				Type:    "string",
 				Default: "foo",
 			},
-			"required-param-not-applicable": {
+			"required-param-with-default-not-applicable": {
```

Comment:
Was this an intended change? I don't see a default set for this definition?

Reply:
🤦 Not intended; thank you! Should be fixed now.